Modules/api_handler.py

```diff
@@ -80,3 +80,19 @@ def __init__(self, hostname: str, token: str=None, tls=False):
             await self._connection.send(data)
         else:
             await self._connection.send(json.dumps(data))
+
+    def construct_request(self, endpoint: (str, str), root_params: {} = {}, meta_params: {} = {}) -> str:
+        """ Constructs the JSON to be passed along the WebSocket channel from the given parameters
+            root_params has the power to overwrite the endpoint!
+            Params:
```

Comment:
Google's format uses `Arguments:` or `Args:`. Those two are also parsed by PyCharm etc. which is nice.